src/CoreBundle/Bridge/Chameleon/Migration/Script/update-1549281421.inc.php

```diff
@@ -219,7 +219,52 @@
 $query = 'ALTER TABLE `cms_menu_category` ADD INDEX ( `position` )';
 TCMSLogChange::RunQuery(__LINE__, $query);
 
-// Add field cms_menu_item
+// Add icon_font_css_class field.
+
+$iconFontCssClassFieldId = TCMSLogChange::createUnusedRecordId('cms_field_conf');
+
+$data = TCMSLogChange::createMigrationQueryData('cms_field_conf', 'en')
+    ->setFields([
+        'cms_tbl_conf_id' => $cmsMenuCategoryTableId,
+        'name' => 'icon_font_css_class',
+        'translation' => '',
+        'cms_field_type_id' => TCMSLogChange::GetFieldType('CMSFIELD_STRING'),
+        'cms_tbl_field_tab' => '',
+        'isrequired' => '0',
+        'fieldclass' => '',
+        'fieldclass_subtype' => '',
+        'class_type' => 'Core',
+        'modifier' => 'none',
+        'field_default_value' => '',
+        'length_set' => '',
+        'fieldtype_config' => '',
+        'restrict_to_groups' => '0',
+        'field_width' => '',
+        'position' => '0',
+        '049_helptext' => '',
```

Comment:
Some short explanations? (on when to use for example)

Reply:
- Added help text.
- Improved help texts for other icon class fields.
- Added German translation where missing.